src/main/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealService.java

```diff
@@ -90,19 +91,28 @@ public Long submitAppeal(SyaCaseWrapper appeal, String userToken) {
 
     @SuppressWarnings("unchecked")
     public Optional<SessionDraft> getDraftAppeal(String oauth2Token) {
-        List<SscsCaseData> caseDetailsList = citizenCcdService.findCase(getUserTokens(oauth2Token));
+        SscsCaseData caseDetails = null;
+        IdamTokens idamTokens = getUserTokens(oauth2Token);
+        List<SscsCaseData> caseDetailsList = citizenCcdService.findCase(idamTokens);
         if (CollectionUtils.isNotEmpty(caseDetailsList)) {
-            SessionDraft sessionDraft = (SessionDraft) convertAintoBService.convert(caseDetailsList.get(0));
+            caseDetails = caseDetailsList.get(0);
+            SessionDraft sessionDraft = (SessionDraft) convertAintoBService.convert(caseDetails);
             return Optional.of(sessionDraft);
         }
+        log.info("GET Draft case with CCD Id {} , IDAM Id {} and roles {} ",
```

Comment:
this log message is only reached when no draft is found and therefore the caseDetails == null check is redundant. There will be no message logged when the draft is found.

Reply:
ah.. good catch let me fix it

Reply:
Fixed now